doc/development/plugins.rst

```diff
@@ -466,11 +466,13 @@ where
 * :attr:`~.Operation.grad_method`: the gradient computation method; ``'A'`` for the analytic
   method, ``'F'`` for finite differences, and ``None`` if the operation may not be differentiated
 
-* :attr:`~.Operation.grad_recipe`: The gradient recipe for the analytic ``'A'`` method.
-  This is a list with one tuple per operation parameter. For parameter :math:`k`, the tuple is of
-  the form :math:`(c_k, s_k)`, resulting in a gradient recipe of
+* :attr:`~.Operation.grad_recipe`: The gradient recipe for the analytic ``'A'``
+  method. This is a tuple with one nested list per operation parameter. For
```

Comment:
Oh, this was because the multiplier and the shift may be mutated in non-tape mode (see PR description for more details):
https://github.com/PennyLaneAI/pennylane/blob/6d489eaedc192155e2b1119bb90a40394fc4fc1f/pennylane/operation.py#L644
Therefore, it made sense to choose a mutable collection rather than a `tuple`.

Reply:
Perfect!